### ISSUE TITLE
Known issues with Mode 10 and placeable bunker silo?

### ISSUE THREAD

**Issue by Th301** [Courseplay/courseplay#6504]
Are there any known issues with the Mode 10 and placeable bunker silos?

Yesterday I tried to set up my Xerion to push in the chaff into a bunker silo (Keilsilo).

I drove in the course like it is described in a lot of videos on YT and in the initial Mode 10 thread (and like I did it many times before). The result is, that the CP driver than drives anywhere but not on the course.

Is it possible that there is an issue with mod silos? Or is it perhaps because there was no chaff in the silo when I wanted to try the course I recorded?

**Comment by Tensuko** [comment]
We have known Issues, and for that we got a search function...

Start inside the Silo, drive to a save place, add a waitpoint, switch to reverse, drive in front of the silo, stop record and done.

If you record it with or without chaff in it doesn't matter. But ofc you will some chaff in it when you start the course.
For more informations read the other Issues pls before you open a new one.

**Comment by Th301** [comment]
I of course used the search function. Filtering for "mode 10" lead to 4 pages. I read al of the issues and nothing explains the behavior. So I created this new one ;)

Like I wrote: I recorded the course exactly like you described. (I already participated in the initial post of the release of mode 10.)
It was late yesterday. I will do some further investigations today with debug channels.

**Comment by Th301** [comment]
One little addition. (Perhaps it can be helpful once)

I tried again. Recorded the course again. Always the same thing. (Even with different vehicles). Tractor was not going to drive on the course.

I found that it seems to be a problem with "aufschütten". I switched to "verteilen und verdichten", started the same course again and it worked. Than I switched back to "aufschütten" and it also worked!

Now the corn/silage season can begin ;)
